webhook/certificates/controller.go

```diff
@@ -68,7 +70,7 @@ func NewController(
 
 	// Reconcile when the cert bundle changes.
 	secretInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
-		FilterFunc: controller.FilterWithNameAndNamespace(system.Namespace(), wh.secretName),
+		FilterFunc: controller.FilterWithNameAndNamespace(key.Namespace, key.Name),
```

Comment:
lol the func name `FilterWithNameAndNamespace` doesn't match the args order

Reply:
Yeah, and it'd be nice if it took a `types.NamespacedName` 🙄